### ISSUE TITLE
Documentation minor typo in configuration

### ISSUE THREAD

**Issue by moxious** [neo4j-contrib/neo4j-streams#273]
In the documentation here: 

https://neo4j.com/docs/labs/neo4j-streams/current/docker/#_source_module

There is a bit of config that looks like this:

```- ./neo4j/plugins:/plugins:/plugins```

Which should be this:

```- ./neo4j/plugins:/plugins```

**Comment by topher235** [comment]
Not sure if I should open a new issue since these could all be fixed at the same time and they all fall under the umbrella of "documentation."

1. The neo4j image in the `docker-compose.yml` is `image: neo4j:3.4`. This did not work for me. When running `CALL streams.publish('neo4j', 'hello world');` I got this error message:

```
Failed to invoke procedure 'streams.publish': Caused by: kotlin.UninitializedPropertyAccessException: lateinit property eventRouterConfiguration has not been initialized
```

My fix was to change the image to `image: neo4j:3.5`.

2. The documentation states that by running the command that generates random data, the consumer will show output. This was not the case. I had to add the following environment variables to the `docker-compose.yml`:

```
NEO4J_streams_source_topic_nodes_neo4j: Person{*}
NEO4J_streams_source_topic_relationships_neo4j: KNOWS{*}
```
